core/src/dispatch/lookup/gc.rs

```diff
@@ -34,6 +34,7 @@ pub struct UpdateStrategy {
     algo: FeedbackAlgorithm,
 }
 
+#[allow(dead_code)]
```

Comment:
This shouldn't be necessary anymore, since I added the from_config constructor to create those cases.